site/pages/disruption-detail/[disruptionId].page.tsx

```diff
@@ -655,20 +655,23 @@ const DisruptionDetail = ({
                                 </div>
                             ))}
                         </div>
-                        {disruption.socialMediaPosts && disruption.socialMediaPosts.length < 5 ? (
+
+                        <button
```

Comment:
Same point here around using Links like in the other PR

Reply:
done